doc/en/components/grids/_shared/row-adding.md

```diff
@@ -287,49 +401,92 @@ After a new row is added through the row adding UI, its position and/or visibili
 
 <!-- ComponentEnd: HierarchicalGrid -->
 
-<!-- Angular -->
 ## Customizing Row Adding Overlay
 
 ### Customizing Text
 
 Customizing the text of the row adding overlay is possible using the `RowAddTextDirective`.
 
+<!-- Angular -->
 ```html
-<ng-template igxRowAddText>
-	Adding Row
-</ng-template>
+<{ComponentSelector} [data]="data" [primaryKey]="'ProductID'" [autoGenerate]="false" [rowEditable]="true">
+    <ng-template igxRowAddText>
+	    Adding Row
+    </ng-template>
+</{ComponentSelector}>
 ```
+<!-- end: Angular -->
 
+<!-- WebComponents -->
 ```ts
-public addRowTextTemplate = (ctx: IgcGridRowEditActionsTemplateContext) => {
+this.grid.rowAddTextTemplate = (ctx: IgcGridEmptyTemplateContext) => {
     return html`Adding Row`;
 }
 ```
+<!-- end: WebComponents -->
+
+```razor
+<{ComponentSelector} Data="data" PrimaryKey="ProductID" AutoGenerate="false" RowEditable="true" RowAddTextTemplate="addTextTemplate">
+</{ComponentSelector}>
+
+@code {
+    private RenderFragment<IgbGridEmptyTemplateContext> addTextTemplate = (context) =>
+    {
+        return @<span>Adding Row</span>;
+    };
+}
+```
 
 ### Customizing Buttons
 
 Customizing the buttons of the row editing overlay is possible using the `RowEditActionsDirective`.
 
 If you want the buttons to be part of the keyboard navigation, then each on of them should have the `RowEditTabStopDirective`.
 
- ```html
- <ng-template igxRowEditActions let-endRowEdit>
+<!-- Angular -->
+```html
+<ng-template igxRowEditActions let-endRowEdit>
 	<button igxButton igxRowEditTabStop (click)="endRowEdit(false)">Cancel</button>
 	<button igxButton igxRowEditTabStop (click)="endRowEdit(true)">Apply</button>
 </ng-template>
 ```
 
+> **Note**:
+> Using `RowEditActions` directive will change edit actions for both editing and adding overlay buttons.
+<!-- end: Angular -->
+
+<!-- WebComponents -->
 ```ts
-public editActionsTemplate = (ctx: IgcGridRowEditActionsTemplateContext) => {
+this.grid.rowEditActionsTemplate = (ctx: IgxGridRowEditActionsTemplateContext) => {
     return html`
         <button onClick="${this.endRowEdit(false)}">Cancel</button>
 	    <button onClick="${this.endRowEdit(true)}">Apply</button>
     `;
 }
 ```
-> [!Note]
-> Using `RowEditActions` directive will change edit actions for both editing and adding overlay buttons.
+<!-- end: WebComponents -->
+
+```razor
+<{ComponentSelector} Data="data" PrimaryKey="ProductID" AutoGenerate="false" RowEditable="true" RowEditActionsTemplate="rowEditActionsTemplate">
+</{ComponentSelector}>
+
+@code {
+    private RenderFragment<IgbGridRowEditActionsTemplateContext> rowEditActionsTemplate = (context) =>
+    {
+        return @<div class="row-actions">
+                    <button onClick="${this.endRowEdit(false)}">Cancel</button>
```

Comment:
Not sure if the ${} syntax would render correctly in Blazor.

Reply:
@skrustev What about `${}` ? Is that working as expeted?